magento/ece-tools
# MAGECLOUD-5306: Run the update command after enabling SPLIT_DB

https://magento2.atlassian.net/browse/MAGECLOUD-5306

### Description
Adds the launch of a CLI command `bin\magento setup:upgrade` after enabling split databases

### Fixed Issues (if relevant)
https://magento2.atlassian.net/browse/MAGECLOUD-5306

### Manual testing scenarios
1. Enable split database in Magento Cloud for any type
    Expected: 
    The splitting went successful
2. Open '<magento_root>/var/log/cloud.log' log file
     Expected: 
     After executing each command for splitting databases, the upgrade command `php bin/magento setup:upgrade ` is executed 

### Contribution checklist
